library/ssl_srv.c

```diff
@@ -1942,6 +2061,19 @@ static int ssl_parse_client_hello( mbedtls_ssl_context *ssl )
                 break;
 #endif /* MBEDTLS_SSL_SESSION_TICKETS */
 
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+            case MBEDTLS_TLS_EXT_USE_SRTP:
+                MBEDTLS_SSL_DEBUG_MSG( 3, ( "found use_srtp extension" ) );
+
+                if ( ssl->conf->transport == MBEDTLS_SSL_TRANSPORT_DATAGRAM )
```

Comment:
As above: This skipping logic would normally happen in the extension handler. Not a blocker.